src/test/ssr.test.js

```diff
@@ -46,6 +46,22 @@ describe('ssr', () => {
     expect(css).toMatchSnapshot()
   })
 
+  it('should respect removed rules', () => {
+    const Heading = styled.h1`
```

Comment:
It’s fine to have a test I guess, but this should never happen. In the SSR use case, you’d never reuse a stylesheet and once render is complete, you’re done.

Reply:
Yea, this is really just to test the remove method, because I wasn’t sure where I had to do the least amount of work to write the test 😛

I’ll see if there’s a better place

Reply:
moved